emukit/quadrature/kernels/quadrature_kernels.py

```diff
@@ -20,18 +21,36 @@ class QuadratureKernel:
     An example of a specific QuadratureKernel and IStandardKernel pair is QuadratureRBF and IRBF.
     """
 
-    def __init__(self, kern: IStandardKernel, integral_bounds: List[Tuple[float, float]], integral_name: str='') \
+    def __init__(self, kern: IStandardKernel, integral_bounds: Union[None, List[Tuple[float, float]]],
+                 measure: Union[None, IntegrationMeasure], variable_names: str= '') \
             -> None:
         """
         :param kern: standard emukit kernel
         :param integral_bounds: defines the domain of the integral. List of D tuples, where D is the dimensionality
         of the integral and the tuples contain the lower and upper bounds of the integral
-        i.e., [(lb_1, ub_1), (lb_2, ub_2), ..., (lb_D, ub_D)]
-        :param integral_name: the (variable) name(s) of the integral
-        """
+        i.e., [(lb_1, ub_1), (lb_2, ub_2), ..., (lb_D, ub_D)]. None if bounds are infinite
+        :param measure: the integration measure. Note that the integral in the QuadratureKernel are specific to the
+        measure. None if no measure.
+        :param variable_names: the (variable) name(s) of the integral
```

Comment:
What are they for?

Reply:
what do you mean with "they"? It is not clear from the comment.

Reply:
i see. if github showed only 1 line it would be clearer. i talking about "variable_names" parameter

Reply:
It is a string that gives the user the option to describe the variables with words. For instance I used the BQ model in the SEIR application and forgot which dimension stands for which physical parameter, e.g., dim0->infection rate, etc. A string is just a minimal version to add some kind of description in case someone is as forgetful as me. It is not used for anything inside the code.